src/main/java/tarawrr/Parser.java

```diff
@@ -0,0 +1,69 @@
+package tarawrr;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ ** Parser Class - Responsible for parsing user input commands.
+ */
+public class Parser {
+
+    /**
+     * Parse user input into its respective command and throws an exception if invalid input is given
+     * @param input
+     * @return
+     * @throws TarawrrException
+     */
+    public static Command parseTask(String input) throws TarawrrException {
+        String taskType = input.split(" ")[0].toLowerCase().trim();
+
+        switch (taskType) {
+        case "todo":
+            String description = input.substring(taskType.length()).trim();
+            if (description == "") {
```

Comment:
```suggestion
            if (description.isEmpty()) {
```

For checking if the `description` string is empty, would you consider using `description.isEmpty()` instead of `description == ""`? I find it's a bit more explicit about the intent to check for an empty string.